php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -66,6 +66,15 @@ public static boolean isStaticClassMember(List<AstNode> modifiers) {
     return false;
   }
 
+  public static boolean hasModifier(List<SyntaxToken> modifiers, String toFind) {
```

Comment:
@pynicolas May be rename it to "contains"? 
And why not put this method to check implementation (as it's used only there)?